egs/librispeech/ASR/pruned_transducer_stateless/train.py

```diff
@@ -186,6 +197,30 @@ def get_parser():
         help="The seed for random generators intended for reproducibility",
     )
 
+    parser.add_argument(
+        "--save-every-n",
+        type=int,
+        default=8000,
+        help="""Save checkpoint after processing this number of batches"
+        periodically. We save checkpoint to exp-dir/ whenever
+        params.batch_idx_train % save_every_n == 0. The checkpoint filename
+        has the form: f'exp-dir/checkpoint-{params.batch_idx_train}.pt'
+        Note: It also saves checkpoint to `exp-dir/epoch-xxx.pt` at the
+        end of each epoch where `xxx` is the epoch number counting from 0.
+        """,
+    )
+
+    parser.add_argument(
+        "--keep-last-k",
+        type=int,
+        default=10,
```

Comment:
I suspect we might want to average more than 10, typically (depending how frequently they are saved).

Reply:
10 is just the default value and can be provided by users from the command line.
What default value do you recommend here?

Reply:
I'm thinking maybe 20.